src/shared/lib/resources/opportunity/sprint-with-us.ts

```diff
@@ -123,9 +132,12 @@ export type CreateSWUOpportunityStatus
   | SWUOpportunityStatus.UnderReview
   | SWUOpportunityStatus.Draft;
 
-export interface CreateSWUOpportunityPhaseBody extends Omit<SWUOpportunityPhase, 'createdAt' | 'createdBy' | 'updatedAt' | 'updatedBy' | 'startDate' | 'completionDate'> {
+export type CreateSWUOpportunityPhaseRequiredCapabilityBody = Omit<SWUOpportunityPhaseRequiredCapability, 'createdAt' | 'createdBy' | 'updatedAt' | 'updatedBy'>;
```

Comment:
More a nitpick than anything — using a `Pick` here would be much easier to read: `Pick<SWUOpportunityPhaseRequiredCapability, 'capability' | 'fullTime'>`

Reply:
Optional change

Reply:
Good suggestion, thanks.  Done.